agrirouter-api-java-api/src/main/kotlin/com/dke/data/agrirouter/api/service/parameters/SecuredRegistrationRequestParameters.kt

```diff
@@ -13,4 +13,5 @@ class SecuredRegistrationRequestParameters : ParameterValidation {
 
     lateinit var redirectUri: String
 
+    var state: String = ""
```

Comment:
Do you need to initialize an empty variable? Couldn't w use a `lateinit` here as well?

Reply:
changed for State, as we don't necessarily need a default Value.
But also redirectUri was changed to that, as it needs a default value to avoid nullPointerExceptions